baekjoon/number_theory/1929.js

```diff
@@ -0,0 +1,39 @@
+const getPrimes = () => {
+  const BOUNDARY = 1000001;
+  const count = Array(BOUNDARY).fill(false);
+
+  for (let i = 2; i < BOUNDARY; i++) {
+    if (count[i]) continue;
+
+    for (let j = 2 * i; j < BOUNDARY; j += i) {
+      count[j] = true;
+    }
+  }
+
+  return count
+    .map((isNotPrime, i) => (isNotPrime ? 0 : i))
+    .filter(v => v > 0)
+    .slice(1);
+};
+
+const solution = (M, N) => {
```

Comment:
solution 은 문제에 대한 컨텍스트를 담고 있기 때문에 변수 이름에 의미를 부여하는 것도 좋을 것 같습니다.
```suggestion
const solution = (start, end) => {
```

Reply:
다음부터 반영해보겠습니다 ㅎㅎ 감사합니다.